js/aside.js

```diff
@@ -29,14 +29,14 @@ export const renderAside = (currentDate, updateDate) => {
   };
 
   const renderPreviousMonthDays = (startDay, previousMonthTotalDays) => {
-    for (let i = 0; i < startDay; i++) {
-      const day = previousMonthTotalDays - startDay + 1 + i;
+    for (let i = 0; i < startDay - 1; i++) { 
```

Comment:
this should be improved. startDay is already calculated before passing to renderPreviousMonthDays and then again minus one happens. unclear what is happening